packages/replay-next/src/suspense/EventsCache.ts

```diff
@@ -110,27 +111,52 @@ async function fetchEventTypeEntryPoints(
   eventType: EventHandlerType,
   record: Record<EventLog[]>
 ) {
-  const entryPoints = await client.runAnalysis<EventLog>({
-    effectful: false,
-    eventHandlerEntryPoints: [{ eventType }],
-    mapper: getFunctionBody(eventsMapper),
-  });
-
-  // Pre-cache object previews that came back with our new analysis data.
-  // This will avoid us having to turn around and request them again when rendering the logs.
-  entryPoints.forEach(entryPoint => cachePauseData(client, entryPoint.pauseId, entryPoint.data));
-
-  const eventLogs: EventLog[] = entryPoints.map(entryPoint => ({
-    ...entryPoint,
-    type: "EventLog",
-  }));
-
   const wakeable = record.value;
+  try {
+    const entryPoints = await client.runAnalysis<EventLog>({
+      effectful: false,
+      eventHandlerEntryPoints: [{ eventType }],
+      mapper: getFunctionBody(eventsMapper),
+    });
 
-  record.status = STATUS_RESOLVED;
-  record.value = eventLogs;
-
-  wakeable.resolve(eventLogs);
+    // Pre-cache object previews that came back with our new analysis data.
+    // This will avoid us having to turn around and request them again when rendering the logs.
+    entryPoints.forEach(entryPoint => cachePauseData(client, entryPoint.pauseId, entryPoint.data));
+
+    const eventLogs: EventLog[] = entryPoints.map(entryPoint => ({
+      ...entryPoint,
+      type: "EventLog",
+    }));
+
+    record.status = STATUS_RESOLVED;
+    record.value = eventLogs;
+  } catch (err) {
+    // Handle any analysis errors (such as "too many hits found") by resolving the record with
+    // a dummy `EventLog` entry that will get shown as a single console message at the top of
+    // the list.  That way we at least communicate to the user what happened.
+    // TODO [FE-1257] Show a better indicator that there were too many events found, like an overlay.
+    record.status = STATUS_RESOLVED;
+    record.value = [
+      {
+        data: {
+          frames: [],
+          objects: [],
+        },
+        location: [],
+        pauseId: "",
+        point: "",
+        time: 0,
+        type: "EventLog",
+        values: [
+          {
+            value: `❗Too many messages of type "${eventType}" found!`,
```

Comment:
The embedded "❗" character feels kind of yucky. Especially the lack of a space between "❗" and the subsequent "T" character. I don't think we should mix iconography with error message.

Reply:
Can we remove this bit?